src/business-logic/services/ingest-data-changed-service.ts

```diff
@@ -254,6 +277,8 @@ export class IngestDataChangedService implements DataChangeService {
     }
     const eventCallback: (() => Promise<void>) | undefined = this.getEventToExecute()
     if (!eventCallback) {
+      void this.generateActions()
+
```

Comment:
Superfluous newline here, in my opinion

Reply:
Since you've put it so nicely, I have removed it ;)